app/istqb-ctel-tm-sm-complete-guide/istqb-ctel-tm-sm-complete-guide.css

```
@@ -0,0 +1,740 @@
+/* ===== KEYFRAMES ===== */
+@keyframes pulse {
+  0%, 100% { transform: translate(-50%,-50%) scale(1); opacity: 0.6; }
+  50% { transform: translate(-50%,-50%) scale(1.15); opacity: 1; }
+}
+@keyframes fadeInDown {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fadeInDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)